CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.1.20
+
+* Upgrade Indexer Framework to be based on [nearcore 1.31.0](https://github.com/near/nearcore/commit/1c6fa351a6f285734835cc1e7c9f0a03e38ca49b)
```

Comment:
I guess it's worth to be upgraded to 1.31.1 already, would you mind?

Reply:
This PR wasn't meant to include this commit, I forgot to point it to #62. I'll merge this PR just with the CI fixes and create a new PR for `1.31.1`.